aws/resource_aws_organization.go

```diff
@@ -0,0 +1,109 @@
+package aws
+
+import (
```

Comment:
Nitpick: can you please structure the imports as:
```go
import (
  // stdlib ones

  // others
)
```

FYI many folks here are using [`goimports`](https://godoc.org/golang.org/x/tools/cmd/goimports) to automatically add, remove, and format imports 😄